### ISSUE TITLE
loading pre-trained model

### ISSUE THREAD

**Issue by nikparth** [zyxxmu/OptG#1]
Hi I'm trying to load the pre-trained model and there are keys for the bias mask (".b_mask") but this is not in the model code so there are incompatible keys. I'm assuming this is an error but if not please comment.

Thank you.

**Comment by zyxxmu** [comment]
We apologize for the inconvenience caused by the mismatch between the training code used to generate the checkpoint and the current code version. We have fixed this bug, and we appreciate your attention to our work. If you have any further questions, please feel free to discuss.

**Comment by nikparth** [comment]
thank you for addressing.
